### ISSUE TITLE
Not able to login to firebase 

### ISSUE THREAD

**Issue by ajaysagar99** [google/android-emulator-container-scripts#364]
After doing all the necessary steps, I'm stuck at this screen -> 

This screen opens in HTTP (not HTTPS) and click does not lead to signin at all 

Ubuntu 22

emu-docker create stable "P.*x86_64"

./create_web_container.sh

docker-compose -f js/docker/docker-compose-build.yaml -f js/docker/development.yaml up

<img width="1440" alt="Screenshot 2023-11-17 at 2 39 52 PM" src="https://github.com/google/android-emulator-container-scripts/assets/12610508/0b27d96d-4d14-4e0f-b779-1ce48371e6eb">

More suggestions - 

1. Step 1.2 in Setting up AAOS Emulator Service needs to be - cd android-emulator-container-script`s`
2. ./create_web_container.sh -p user1,passwd1 - -p user1, pwd1 doesn't seem to work with this script, kindly check 

Thanks in advance, cheers!

**Comment by ajaysagar99** [comment]
<img width="1440" alt="Issue" src="https://github.com/google/android-emulator-container-scripts/assets/12610508/56ebdd86-5748-4b8a-bf14-cb34be56abb6">

After some debugging I am seeing this issue. 

The domain is already whitelisted in Firebase also not sure what is wrong

**Comment by jpcottin** [comment]
@ajaysagar99 — the underlying setup has changed substantially since you filed this. Two of the three things in your report should now be in a better place; the third (the `-p` flag) needs a separate issue.

**Login screen / Firebase sign-in not working** — likely fixed. The bundled `firebase_config.json` pointed at `android-emulator-webrtc-demo`, a Firebase project nobody on the team has had access to for some time, so Firebase auth was broken-by-default. After #408 (v11 modular SDK migration) and #409 (BYO Firebase restructure, both merged today), the setup is now documented per-contributor — see [js/README.md → Authentication](https://github.com/google/android-emulator-container-scripts/blob/master/js/README.md#authentication). One caveat: if you're accessing the UI over plain HTTP from a non-localhost address, Firebase popup-based sign-in won't work (Firebase requires HTTPS or localhost). For deployed setups, the [self-hosted JWT path](https://github.com/google/android-emulator-container-scripts/blob/master/js/README.md#self-hosted-jwt-recommended-for-deployments) is the recommended option.

**Doc typo** (extra backtick after `container-script`) — small. The README has changed since this was filed, so please re-check on current ToT and open a tiny issue or PR if it's still there.

**`create_web_container.sh -p user1,passwd1` not working** — separate from the Firebase issue. If you can still reproduce on current ToT, please open a new issue with the exact command and the error/behavior.

Closing this one as the primary Firebase symptom should be resolved by the new BYO setup. Please open new issues for any of the above that still reproduces — the code and instructions have changed substantially since this was filed, so fresh threads will be easier to track.
